data_loaders/json_loader.py

```diff
@@ -0,0 +1,16 @@
+from abc import ABC 
+from data_loaders.data_loader import DataLoader
+import json
+
+class JsonLoader(DataLoader):
+	def __init__(self, file_name):
+		self.file = file_name
+	def load(self):
+		with open(self.file) as f:
+  			data = json.load(f)
+		r = []
```

Comment:
I only did this to save the trouble of converting into a pandas dataframe in the class, since all I need in the driver is what I'm returning here. If I converted to a pandas dataframe here, all I would do is change it back to an array of data in the driver class anyway.